explainer/Explainer.md

```diff
@@ -1,9 +1,178 @@
+especially relevant.  The WoT Discovery specification also includes a Thing Model,
+defined in this deliverable,
+which normatively defines many aspects of the Thing Description Directory service specified
+as part of the discovery process.  
+
+## Use Cases and Requirements
+In general, WoT Use Case and Requirements have been collected from a large number 
+of participants and are collected in the 
+[WoT Use Cases and Requirements document](https://w3c.github.io/wot-usecases/).
+In the following we extract a few of the most critical requirements and motivating
+use cases driving the design of WoT Discovery.
+
+### Integration
+It is often the case in IoT systems that devices provided by different manufacturers
+need to be integrated together.  For example, consider a Retail use case
+where a door sensor, a temperature sensor, and an RGB light need to be integrated
+with an automation to indicate when a freezer door has been left open or the 
+temperature is out of range.  As another example, we might want to integrate an
+accessibility control, such as a blow switch, with an IoT device that operates a 
+light or calls an elevator.  In these cases, it is not enough to have a common
+interface description format, which the WoT TD provides.  We also need a standard
+way to find and fetch the TDs for specific devices so that they can be integrated.
+
+### Public IoT Services
+In a Smart City, we may want to provide access to IoT devices as a public service,
+for example to report local air quality conditions.  In Retail, we may want to
+make available semi-public services (perhaps requiring pre-registration) to access
+accessiblity devices, such as chair lifts.  In this case, a user wants to discover
+services, probably selected from a large number, that meet some criteria, such
+as location or type.
+
+### Privacy and Security
+While the purpose of the WoT Discovery process is to distribute Thing Description metadata,
+it must also satisfy security and privacy constraints.
+In general, TDs should only be distributed to entities after their identities have been
+confirmed (authentication) and it has been established that they have the rights to view that data
+(authorization).  In addition, data needs to be protected from interception and modification,
+and it is also necessary to guard against other risks such as tracking or inferencing of
+private data.  
+
+The WoT Discovery architecture has been defined with safeguarding security and privacy 
+as a primary goal.
+
+The WoT Architecture and WoT Thing Description documents have additional security and
+privacy considerations that are relevant to WoT Discovery.  In particular, 
+the WoT Architecture document discusses when and where secure transport such as TLS
+should be used, and when various access control mechanisms should be used.
+The problem is complicated by the fact that we often want to access IoT devices
+on a LAN where the CA system available on the rest of the internet is not easy
```

Comment:
I think some abbreviations like CA could/should be spelled out the first time they are being used (although the explainer's target audience is probably familiar with the terminology).

Reply:
yeah, I was thinking that too.  Expanded.